doc/development/development.rst

```diff
@@ -151,23 +151,70 @@ navigate to your local repository in your terminal and run the following command
 
     pre-commit install
 
-Now, each time you make a commit, pre-commit will automatically run black and any other configured hooks.
+Now, each time you make a commit, pre-commit will automatically run Black and any other configured hooks.
 If the hooks make changes or if there are any issues, the commit will be stopped, and you'll be able to review and add the changes.
 
-If you want black to omit a line from formatting, you can add the following comment to the end of the line:
+If you want Black to omit a line from formatting, you can add the following comment to the end of the line:
+
+.. code-block:: python
+
+    # fmt: skip
+
+To ignore a block of code you must flank the code with two comments:
 
 .. code-block:: python
 
     # fmt: off
+    code here
+    # fmt: on
 
 As described in the `black documentation <https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#code-style>`_,
 
-The following are some styling conventions that we follow in SpikeInterface:
+
```

Comment:
Let's make a small section here that is "doctring conventions" or something like this.

Reply:
Done :)